### DISCUSSION TITLE
Tables not find in Flink SQL Client

### DISCUSSION THREAD

**Discussion by xxubai** [apache/amoro#502] [General]
Hi team.
AMS(arctic163/ams) image was deployed in our k8s pod, and running smoothly. Next, I start the flink(arrctic163/flink) image in local environment and create catalog base on your documentation.
But when I run below instructions it throws `org.apache.flink.table.api.ValidationException: Tables or views with the identifier 'arctic.db_name.table_name' doesn't exist`
```
desc table_name;
```
OR
```
INSERT INTO arctic.db_name.table_name VALUES (...)
```

Otherwise, ams service works fine when running locally.

**Comment by xxubai** [comment]
![05_36_28](https://user-images.githubusercontent.com/54210725/196919267-2d7a7731-8769-4abf-81bd-73f314653b9f.jpg)

**Comment by zhoujinsong** [comment]
@XBaith Hi, Thanks for trying Arctic!

AMS(arctic163/ams) image use a local catalog for default in its config file `conf/config.yaml`.
Local catalog store all data files in local file system，So your local flink client may not access data files of tables.
You could add a new hadoop catalog / hive catalog with this guide:https://arctic.netease.com/ch/meta-service/dashboard/#hadoop

So all the clients and AMS distributed in different machines can access to all tables.
